src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
It is necessary to break this method into 3 methods.
1. Reading data from a file
2. Create a report
3. Write the report to a file
This method should consist of calling these methods:
String data = readFromFile(fromFileName);
String report = createReport(data);
writeToFile(report, toFileName);